packages/odyssey-stylelint/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "stylelint": "^9.3.0",
     "stylelint-checkstyle-formatter": "^0.1.1",
+    "stylelint-config-property-sort-order-smacss": "^7.1.0",
     "stylelint-config-recommended": "^2.1.0",
     "stylelint-config-sass-guidelines": "^5.0.0",
```

Comment:
non-blocking: If you can safely update to [`^8.0.0`](https://github.com/bjankord/stylelint-config-sass-guidelines/releases/tag/v8.0.0), we'll avoid multiple versions of the nested dependency `stylelint-order`.

Reply:
we may not need the direct dep anymore - i'll remove and test.

Reply:
after a bit of investigation, upgrading here creates additional nested dep versions of other packages.

some of these are so far behind (stylelint is currently @ 13), it might be worth batch upgrading all of our linting packages at this point.

@jeffbelser-okta any thoughts?